_studio/shared/include/mfx_utils.h

```diff
@@ -175,6 +175,9 @@ inline mfxStatus CheckAndDestroyVAbuffer(VADisplay display, VABufferID & buffer_
 }
 #endif
 
+#define MFX_CHECK_EQUALITY_FIELD(Field) l.Field == r.Field
```

Comment:
I think that's too difficult macro name. Maybe MFX_CHECK_FIELDS_EQ or just MFX_FIELDS_EQ (mind plural).

Reply:
@dvrogozh Well, I'm ok to shorten the name. But I think clarification "FIELDS" should go last, because we have similar macro for "ARRAY". By the way, why plural? This one macro for one field only

Reply:
@dvrogozh how about ```MFX_EQ_FIELD``` and ```MFX_EQ_ARRAY``` ?